remsfal-service/src/main/java/de/remsfal/service/entity/dao/ChatMessageRepository.java

```diff
@@ -0,0 +1,127 @@
+package de.remsfal.service.entity.dao;
+
+import de.remsfal.service.entity.dto.ChatMessageEntity;
+import de.remsfal.service.entity.dto.ChatSessionEntity;
+import de.remsfal.core.model.project.ChatMessageModel.ContentType;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import io.quarkus.panache.common.Parameters;
+
+
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@ApplicationScoped
+public class ChatMessageRepository extends AbstractRepository<ChatMessageEntity> {
+
+    @Inject
+    ChatSessionRepository chatSessionRepository;
+
+    public ChatMessageEntity findChatMessageById(String messageId) {
+        return find("id = :id", Parameters.with("id", messageId))
+                .firstResultOptional()
+                .orElseThrow(() -> new NoSuchElementException("ChatMessage with ID " + messageId + " not found"));
+    }
+
+    @Transactional
+    public ChatMessageEntity sendChatMessage(String sessionId, String senderId, ContentType contentType, String content) {
+        if (sessionId == null || sessionId.isBlank()) {
```

Comment:
This checks could be validated by BeanValidation in the boundy or on the entity.